python_files/nivo_outcomes.py

```diff
@@ -139,24 +70,24 @@
 #                                         variable_col='Time_to_progression_weeks_RECIST1.1', output_dir=current_plot_dir, min_score=0.95)
 
 # summarize hits from all comparisons
-total_dfs = pd.concat(total_dfs)
-total_dfs['log10_qval'] = -np.log10(total_dfs.fdr_pval)
+ranked_features_df = pd.concat(total_dfs)
```

Comment:
I kept `total_dfs` as the list accumulator, but once they're combined together I switched the name to ranked_features_df to be more informative. Same with the output file name.